integration/mosaic-config-injectors/lib/babel/add-aliases.js

```diff
@@ -1,18 +1,49 @@
 const { alias } = require('./util/alias');
 
+const hasPlugin = (pluginName) => (plugin) => {
+    if (Array.isArray(plugin)) {
+        return plugin[0] === pluginName;
+    }
+
+    if (typeof plugin === 'string') {
+        return plugin === pluginName;
+    }
+};
+
+const babelPluginModuleResolverResolvedPath = require.resolve('babel-plugin-module-resolver');
+const babelPluginModuleResolverWithConfig = [
+    babelPluginModuleResolverResolvedPath,
+    {
+        root: 'src',
+        loglevel: 'silent',
+        alias
+    }
+];
+
 const addAliases = (babelConfig) => {
     if (!babelConfig.plugins) {
         babelConfig.plugins = [];
     }
 
-    babelConfig.plugins.push([
-        require.resolve('babel-plugin-module-resolver'), 
-        {
-            root: 'src',
-            loglevel: 'silent',
-            alias
+    if (babelConfig.plugins.some(hasPlugin(babelPluginModuleResolverResolvedPath))) {
+        const babelPluginModuleResolver = babelConfig.plugins.find(hasPlugin(require.resolve));
```

Comment:
require.resolve is a function, I don't see any support for fn arguments in the hasPlugin function

Reply:
oops, typo, my bad

Reply:
fixed